ansible/PKGBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: Alexandre Ferreira < alex dot jorge dot m at gmail dot com >
+
+pkgname=ansible
+pkgver=2.9.7
+pkgrel=1
+pkgdesc='Radically simple IT automation platform'
+arch=('i686' 'x86_64')
+url='https://www.ansible.com'
+license=('GPL3')
+depends=('python3' 'python-yaml' 'python-jinja')
+provides=('python-ansible')
+optdepends=('sshpass: for ssh connections with password')
+makedepends=('tar' 'gcc' 'python3' 'python3-pip' 'libffi' 'libffi-devel' 'libcrypt' 'libcrypt-devel' 'openssl' 'openssl-devel')
+backup=('etc/ansible/ansible.cfg')
+source=("https://releases.ansible.com/ansible/ansible-${pkgver}.tar.gz")
+sha512sums=('ce029441bcafdc5b44c9fda69f183d4defea84ead5628164caf87306cb97efec68c11b2cce728e90f28290640c320549486a6b4e823710f638d1d2e7c35675a4')
+noextract=("${pkgname}-${pkgver}.tar.gz")
+
+prepare() {
+  [[ -d ${pkgname}-${pkgver} ]] && rm -rf ${pkgname}-${pkgver}
+  export MSYS=winsymlinks:lnk
```

Comment:
Related to #1216. This work for me, any better solution???

Reply:
@Alexpux exporting the MSYS variable will not affect the rest of the environment?

Reply:
@dont do that. Remove this export

Reply:
@Alexpux removing `export MSYS=winsymlinks:lnk` lead to this errors extracting source, the better solution is to tar two times as mentioned here #1216?

```
tar: ansible-2.9.7/lib/ansible/modules/web_infrastructure/_jenkins_job_facts.py: Cannot create symlink to ‘jenkins_job_info.py’: No such file or directory
tar: ansible-2.9.7/lib/ansible/plugins/callback/osx_say.py: Cannot create symlink to ‘say.py’: No such file or directory
tar: ansible-2.9.7/lib/ansible/plugins/doc_fragments/ovirt_facts.py: Cannot create symlink to ‘ovirt_info.py’: No such file or directory
tar: ansible-2.9.7/lib/ansible/plugins/lookup/_openshift.py: Cannot create symlink to ‘k8s.py’: No such file or directory
tar: ansible-2.9.7/test/integration/targets/supervisorctl/tasks/install_Darwin.yml: Cannot create symlink to ‘install_pip.yml’: No such file or directory
tar: ansible-2.9.7/test/integration/targets/supervisorctl/tasks/install_FreeBSD.yml: Cannot create symlink to ‘install_pip.yml’: No such file or directory
tar: ansible-2.9.7/test/integration/targets/supervisorctl/tasks/install_RedHat.yml: Cannot create symlink to ‘install_pip.yml’: No such file or directory
tar: ansible-2.9.7/test/integration/targets/supervisorctl/tasks/install_Suse.yml: Cannot create symlink to ‘install_pip.yml’: No such file or directory
tar: ansible-2.9.7/test/integration/targets/supervisorctl/tasks/uninstall_Darwin.yml: Cannot create symlink to ‘uninstall_pip.yml’: No such file or directory
tar: ansible-2.9.7/test/integration/targets/supervisorctl/tasks/uninstall_FreeBSD.yml: Cannot create symlink to ‘uninstall_pip.yml’: No such file or directory
tar: ansible-2.9.7/test/integration/targets/supervisorctl/tasks/uninstall_RedHat.yml: Cannot create symlink to ‘uninstall_pip.yml’: No such file or directory
tar: ansible-2.9.7/test/integration/targets/supervisorctl/tasks/uninstall_Suse.yml: Cannot create symlink to ‘uninstall_pip.yml’: No such file or directory
tar: ansible-2.9.7/test/lib/ansible_test/_data/injector/ansible: Cannot create symlink to ‘python.py’: No such file or directory
tar: ansible-2.9.7/test/lib/ansible_test/_data/injector/ansible-config: Cannot create symlink to ‘python.py’: No such file or directory
tar: ansible-2.9.7/test/lib/ansible_test/_data/injector/ansible-connection: Cannot create symlink to ‘python.py’: No such file or directory
tar: ansible-2.9.7/test/lib/ansible_test/_data/injector/ansible-console: Cannot create symlink to ‘python.py’: No such file or directory
tar: ansible-2.9.7/test/lib/ansible_test/_data/injector/ansible-doc: Cannot create symlink to ‘python.py’: No such file or directory
tar: ansible-2.9.7/test/lib/ansible_test/_data/injector/ansible-galaxy: Cannot create symlink to ‘python.py’: No such file or directory
tar: ansible-2.9.7/test/lib/ansible_test/_data/injector/ansible-inventory: Cannot create symlink to ‘python.py’: No such file or directory
tar: ansible-2.9.7/test/lib/ansible_test/_data/injector/ansible-playbook: Cannot create symlink to ‘python.py’: No such file or directory
tar: ansible-2.9.7/test/lib/ansible_test/_data/injector/ansible-pull: Cannot create symlink to ‘python.py’: No such file or directory
tar: ansible-2.9.7/test/lib/ansible_test/_data/injector/ansible-test: Cannot create symlink to ‘python.py’: No such file or directory
tar: ansible-2.9.7/test/lib/ansible_test/_data/injector/ansible-vault: Cannot create symlink to ‘python.py’: No such file or directory
tar: ansible-2.9.7/test/lib/ansible_test/_data/injector/importer.py: Cannot create symlink to ‘python.py’: No such file or directory
tar: ansible-2.9.7/test/lib/ansible_test/_data/injector/pytest: Cannot create symlink to ‘python.py’: No such file or directory
tar: Exiting with failure status due to previous errors
==> ERROR: A failure occurred in prepare().
```